examples/TemperatureAlert/TemperatureAlert.ino

```diff
@@ -0,0 +1,49 @@
+/*
+  HTS221 - Temperature Alert
+  This example reads data from the on-board HTS221 sensor of the
+  Nano 33 BLE Sense. If the temperature increases above a certain limit, it turns the buzzer on.
+  The circuit:
+  - Arduino Nano 33 BLE Sense
+  - Active buzzer module connected at pin 9
+
+  written by K.Abhijeet
+  This example code is in the public domain
+*/
+
+#include <Arduino_HTS221.h>
+
+float tempLimit = 37;                         // set your temperature limit in °C
```

Comment:
```suggestion
float tempLimit = 37; // set your temperature limit in °C
```
Why these huge random spaces? It just makes it more likely the comment will be off the screen.